pkg/envoy/route_test.go

```diff
@@ -12,14 +12,22 @@ func notIsType(a, b interface{}) bool {
 	return reflect.TypeOf(a) != reflect.TypeOf(b)
 }
 
+func notIsPath(a, b string) bool {
```

Comment:
is this function necessary? 

Could we just compare directly, e.g. `if pathroute != ingress.Spec.Rules[0].HTTP.Paths[0].Path`?  

That seems easier to read to me, now I had to look what this function does to find out it's only a comparison

Reply:
I guess it would also help if the function name was more descriptive like `isNotEqual` or something, but I would still prefer to use the comparison operator if possible

Reply:
Thanks for suggestion @pppontusw 
I removed the comparison function